status/backend.json

```diff
@@ -119,10 +119,10 @@
       "tos_url": "https://io.italia.it/app-content/tos_privacy.html?v=4.9"
     },
     "itw": {
-      "enabled": false,
+      "enabled": true,
```

Comment:
Why we still use enabled? Can't we just user a specific min version to disable/enable the feature to all?